proof.go

```diff
@@ -13,7 +13,10 @@ import (
 )
 
 // ErrFailedCompletenessCheck indicates that the verification of a namespace proof failed due to the lack of completeness property.
-var ErrFailedCompletenessCheck = errors.New("failed completeness check")
+var (
+	ErrFailedCompletenessCheck = errors.New("failed completeness check")
+	ErrWrongLeafHashesSize     = errors.New("wrong leafHashes size")
+)
```

Comment:
Sorry can't provide suggestion but I think the previous Go doc should be moved

```go
var (
	// ErrFailedCompletenessCheck indicates that the verification of a namespace proof failed due to the lack of completeness property.
	ErrFailedCompletenessCheck = errors.New("failed completeness check")
	ErrWrongLeafHashesSize     = errors.New("wrong leafHashes size")
)

```

Reply:
Thanks fixed it https://github.com/celestiaorg/nmt/pull/253/commits/b94a414c3fdd9dad617c5ba893f3f9b979792f78